src/main/java/util/ReadWriteRequests.java

```diff
@@ -0,0 +1,58 @@
+package util;
+
+import constants.UrlConstants;
+import org.apache.http.client.methods.CloseableHttpResponse;
+import org.apache.http.client.methods.HttpPost;
+import org.apache.http.client.methods.HttpGet;
+import org.apache.http.entity.ContentType;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.impl.client.HttpClients;
+import org.apache.http.util.EntityUtils;
+import org.apache.http.entity.StringEntity;
+
+public class ReadWriteRequests {
```

Comment:
I want you to implement some basic http client with some generic logic like get, post, put, delete, with logging interceptors, and so on
once you implement it you can just instantiate it inside of this class and compose any request you like